Tests/ImagineEngineTests/Mocks/BundleMock.swift

```diff
@@ -0,0 +1,32 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+@testable import ImagineEngine
+
+final class BundleMock: BundleProtocol {
+    private(set) var resourceNames = Set<String>()
+    var resources = [String : URL]()
+
+    func url(forResource name: String?, withExtension ext: String?) -> URL? {
+        guard let name = name else {
+            return nil
+        }
```

Comment:
🎨 To make the code a bit easier to read, I suggest adding a newline here 😬 (I promise, I will add that linter 😅!)

Reply:
Meanwhile I tried to add that linter in #38 😄